app/services/widget-settings/widget-settings.ts

```diff
@@ -107,8 +107,12 @@ export abstract class WidgetSettingsService<TWidgetData extends IWidgetData> ext
   protected handleDataAfterFetch(data: any): TWidgetData {
 
     // patch fetched data to have the same data format
-
+    if (data.settings.background) data.settings.background_color = data.settings.background.color;
```

Comment:
The Streamboss widget has a lot of `background` fields. Please check there are no conflicts with such global patching